regulations/static/regulations/js/source/router.js

```diff
@@ -23,35 +23,31 @@ else {
             'diff/:section/:baseVersion/:newerVersion': 'loadDiffSection',
             ':section/:version': 'loadSection',
             ':section': 'loadSection',
-            'preamble/:docId': 'loadPreamble',
-            'preamble/:docId/:section': 'loadPreamble'
+            'preamble/:docId/:section': 'loadPreamble',
+            'preamble/:docId/cfr_changes/:section': 'loadCfrChanges'
         },
 
         loadSection: function(section) {
-            var options = {id: section};
-
-            // to scroll to paragraph if there is a hadh
-            options.scrollToId = Backbone.history.getHash();
-
-            // ask the view not to route, its not needed
-            options.noRoute = true;
-
-            MainEvents.trigger('section:open', section, options, 'reg-section');
+          this.openSection(section, 'reg-section');
         },
 
         loadPreamble: function(docId, section) {
-            var parts = [docId];
-            if (section) {
```

Comment:
Yep--previously you could load a url like `/preamble/2016_02749`, but those urls now redirect to the 0th preamble section, like `/preamble/2016_02749/intro`.